dsl/src/main/java/tech/pegasys/peeps/network/Network.java

```diff
@@ -481,6 +482,16 @@ private Map<Web3ProviderType, Genesis> createGenesis(
                   }
                   extraData = new GenesisExtraDataQbft(validators);
                   break;
+                case QBFT_SMART_CONTRACT:
+                  if (e == Web3ProviderType.BESU) {
+                    genesisConfig = new GenesisConfigQbft(chainId, new BftConfig());
+                  } else {
+                    genesisConfig = new GoQuorumQbftConfig(chainId, new BftConfig());
+                  }
+                  genesisConfig.setSmartContractTransition(
```

Comment:
perhaps setValidatorContractValidatorTransaction? It's not clear what the contract is for. And this is for the validatorContract. Could be using privacy contracts for example.